rw-system.sh

```diff
@@ -585,6 +585,9 @@ if getprop ro.vendor.build.fingerprint | grep -iq -e Redmi/rosemary \
     -e Redmi/secret -e Redmi/maltose; then
     setprop debug.sf.latch_unsignaled 1
     setprop debug.sf.enable_hwc_vds 0
+
+    # Exclude FP input devices
+    mount -o bind /system/phh/rosemary-excluded-input-devices.xml /system/etc/excluded-input-devices.xml
```

Comment:
I don't think this will work, have you updated the `PRODUCT_COPY_FILES`?

Reply:
Updated on https://github.com/phhusson/device_phh_treble/pull/325/commits/091321c413955c9d2015ec39871aae4da506548b

Reply:
Have you tested it? `/system/etc/excluded-input-devices.xml` doesn't exist the mount bind will fail